src/main/webapp/WEB-INF/view/searchbar.jsp

```diff
@@ -16,30 +9,63 @@ List<User> allUsers = UserStore.getInstance().getAllUsers();
     return true;
   }
 
-  // Toggle drop down.
-  function toggleDropDown() {
-    var searchBar = document.getElementById("searchBar");
-    var dropDiv = document.getElementById("dropDiv");
-    if (searchBar.value == "") {
-      dropDiv.style.display = "none";
+  // This is to remove the resultItem divs
+  function removeElementsByClass(className){
```

Comment:
Alternatively, you can just set the innerHTML attribute of the result div to an empty string, this removes all children:
`document.querySelector('#result').innerHTML = '';`
This way you don't need the classes (or the IDs) on the result divs.

Reply:
wow, the secrets of javascript. 😂

Reply:
Thanks for this tip, it makes the code a lot cleaner.